calico/_includes/content/calico-windows-install.md

```diff
@@ -0,0 +1,114 @@
+{%- if include.networkingType == "vxlan" %}
+1. Ensure that BGP is disabled since you're using VXLAN.
+   If you installed Calico using operator, you can do this by:
+
+   ```bash
+   kubectl patch installation default --type=merge -p '{"spec": {"calicoNetwork": {"bgp": "Disabled"}}}'
+   ```
+   If you installed Calico using the manifest from https://projectcalico.docs.tigera.io/manifests/calico-vxlan.yaml then BGP is already disabled.
+{%- else %}
+1. Enable BGP service on the Windows nodes (instead of VXLAN).
+   Install the RemoteAccess service using the following Powershell commands:
+
+   ```powershell
+   Install-WindowsFeature RemoteAccess
+   Install-WindowsFeature RSAT-RemoteAccess-PowerShell
+   Install-WindowsFeature Routing
+   ```
+
+   Then restart the computer:
+
+   ```powershell
+   Restart-Computer -Force
+   ```
+
+   before running:
+
+   ```powershell
+   Install-RemoteAccess -VpnType RoutingOnly
+   ```
+   Sometimes the remote access service fails to start automatically after install. To make sure it is running, execute the following command:
+
+   ```powershell
+   Start-Service RemoteAccess
+   ```
+{%- endif %}
+
+1. Download the {{site.prodnameWindows}} installation manifest.
+
+   ```bash
+{%- if include.networkingType == "vxlan" %}
+   curl {{ "/manifests/calico-windows-vxlan.yaml" | absolute_url }} -o calico-windows.yaml
+{%- else %}
+   curl {{ "/manifests/calico-windows-bgp.yaml" | absolute_url }} -o calico-windows.yaml
+{%- endif %}
+   ```
+
+1. Get the cluster's Kubernetes API server host and port, which will be used to update the {{site.prodnameWindows}} config map.
+   The API server host and port is required so that the {{site.prodnameWindows}} installation script can create a kubeconfig file for Calico services.
+   If your Windows nodes already have {{site.prodnameWindows}} installed manually, skip this step. The installation script will
+   use the API server host and port from your node's existing kubeconfig file if the `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT` variables
+   are not provided in the `calico-windows-config` configmap.
```

Comment:
in the `calico-windows-config`config map.

Reply:
I'll use `ConfigMap` to make it clear I'm talking about the kubernetes resource.